config/config.exs

```diff
@@ -7,6 +7,9 @@
 # General application configuration
 use Mix.Config
 
+config :skate,
+  gtfs_url: System.get_env("SKATE_GTFS_URL") || "https://cdn.mbta.com/MBTA_GTFS.zip"
```

Comment:
This works in local development, but won't in production as the configuration is built at compile time.

Instead, you can override the configuration in `Skate.Application.start/2`, which is called at runtime:
```elixir
def start(_, _) do
  override_config!()
end

defp override_config do
  for {application_key, env_var} <- [gtfs_url: "SKATE_GTFS_URL"] do
    if value = System.get_env(env_var) do
      Application.put_env(:skate, application_key, value)
    end
  end
  :ok
end
```

Reply:
Really? Configuring it like this is what Glides does to have different databases for dev and prod. https://github.com/mbta/glides/blob/master/config/prod.exs#L72

Reply:
Glides is deployed differently, as it was built before the deployment code was added. It's not great to leave it like this, but if you do, please add a comment so that future-us knows why the deploys don't pick up the configuration change.

Reply:
[Follow up task](https://app.asana.com/0/1112935048846093/1116881667899749)
I'll do it this sprint.

Reply:
Ended up getting around to this before this PR merged, so I've included it here.